stacks/dovetail.prx.org.yml

```diff
@@ -51,6 +51,9 @@ Parameters:
     Type: String
   DovetailSecretsVersion:
     Type: String
+  DynamodbKinesisArn:
```

Comment:
Okay, so here's what I'm thinking for getting this value into this stack (needs to resolve to `dovetail-ddb-staging` or `dovetail-ddb-production`)...

We already store the stream names in parameter-store.  You can actually see that happening with [serverless/analytics-ingest-lambdas.yml](https://github.com/PRX/Infrastructure/blob/master/stacks/serverless/analytics-ingest-lambdas.yml#L36) pulling the value, out of the key specified in the parent stack [serverless/root.yml](https://github.com/PRX/Infrastructure/blob/master/stacks/serverless/root.yml#L244).

So here, would need these param to have that AWS::SSM type, and the parent stack (root.yml) to interpolate and pass in the SSM param name.